src/entryPoints/SidebarValues.res

```diff
@@ -507,23 +507,13 @@ let systemMetric = permissionJson => {
   })
 }
 
-let paymentSettings = () => {
-  SubLevelLink({
-    name: "Payment Settings",
-    link: `/payment-settings`,
-    access: Access,
-    searchOptions: [("View payment settings", ""), ("View webhooks", ""), ("View return url", "")],
```

Comment:
Can you add these search options to business profile component

Reply:
okay